src/deploy/functions/params.ts

```diff
@@ -590,7 +590,7 @@ async function promptText<T extends RawParamValue>(
   resolvedDefault: T | undefined,
   converter: (res: string) => T | retryInput
 ): Promise<T> {
-  const res = await promptOnce({
+  const res: any = await promptOnce({
```

Comment:
why not `typeof resolvedDefaut | string`? Or just call `resolvedDefautl.toString()` in the prompt call?

Reply:
I didn't want to do the .toString() thing in this call because I wanted as small a footprint of a change in an emergency patch as possible; I'd agree that it'd probably be just as good/better.

The :any annotation is unnecessary with that tip you gave about string.toString() below, since all it's doing is reassuring tsc that it's not going to have to call `never.toString()` down below. I'm getting rid of it.